nevergrad/optimization/test_optimizerlib.py

```diff
@@ -559,3 +559,15 @@ def test_ngo_split_optimizer(name: str, expected: tp.List[str]) -> None:
     opt = optlib.ConfSplitOptimizer(multivariate_optimizer=Opt)(param, budget=1000)
     names = [o.optim.name for o in opt.optims]  # type: ignore
     assert names == expected
+
+@pytest.mark.parametrize(  # type: ignore
+    "dimension,num_workers,expected", [
+    (3, 1, ["SQP", "DiscreteOnePlusOne"]),
+    (1, 20, ["OnePlusOne", "CMandAS2"]),
+    ]
+)
+def test_metangopt8(dimension: int, num_workers: int, expected: tp.List[str]) -> None:
+    param = ng.p.Choice(["const", ng.p.Array(init=list(range(1, dimension+1)))])
+    opt = xpvariants.MetaNGOpt8(param, budget=1000, num_workers=num_workers)
+    names = [o.optim.name for o in opt.optims]  # type: ignore
+    assert names == expected
```

Comment:
I would rather this test was merged with the one above, it's basically with a couple more options + an additional bool for the parametrization

Reply:
Done.